app.py

```diff
@@ -147,14 +152,14 @@ def run_all_tasks():
     list_files()
     delete_file()
     create_user(
-        name + '@test.com',
-        name + '@123',
-        name
+        str(random.random()) + '@test.com',
+        str(random.random()) + '@123',
+        str(random.random())
```

Comment:
I think it would be better and more intuitive to use something like `uuid` instead of `random` as the output generated using random might be weird and it might be same to that of the one created previously.

Reply:
Okay I replace it with uuid

Reply:
@sparkingdark , Don't call `uuid.uuid1` three times. It will generate 3 different values. You need to call `str(uuid.uuid1())` once and store it in a variable lets say `name` and then concat `@test.com` and `@123` to it.